File: Java/Valid_Palindrome.java
void main(){
    StringBuilder a = new StringBuilder();
    String c = IO.readln("Daxil Et: ");
    c = c.toLowerCase();
    for (int i = 0; i < c.length(); i++) {
        if ((c.charAt(i) >= 'a' && c.charAt(i) <= 'z') || (c.charAt(i) >= '0' && c.charAt(i) <= '9'))  {
            a.append(c.charAt(i));
        }
    }

    if (a.toString().equals(a.reverse().toString())) {
        IO.println("Valid Palindrome");        
    }
    else{
        IO.print("Not Valid Palindrome");
    }
    
}

File: Python/Valid_Palindrome.py
def valid_palindrome():
    c = input("Daxil Et: ").lower()
    
    a = ""
    for char in c:
        if ('a' <= char <= 'z') or ('0' <= char <= '9'):
            a += char
            
    if a == a[::-1]:
        print("Valid Palindrome")
    else:
        print("Not Valid Palindrome")

if __name__ == "__main__":
    valid_palindrome()
